reserved_ipv6.go

```diff
@@ -34,6 +34,15 @@ type ReservedIPV6 struct {
 	ReservedAt time.Time `json:"reserved_at"`
 	Droplet    *Droplet  `json:"droplet,omitempty"`
 }
+type ReservedIPV6Resp struct {
+	ReservedIPV6 *ReservedIPV6 `json:"reserved_ipv6"`
```

Comment:
:wave: Hey Ashwani, Thanks for the PR :rocket:. 

Here are some suggestions you may consider:

1. Could you please add `Links` to the ReservedIPV6Resp as it is done [here](https://github.com/digitalocean/godo/blob/96c363fb60e090960d6ed4de5b5494b0e04c1c63/reserved_ips.go#L56-L57)? This will require some changes in Create. You can check [reserved ips](https://github.com/digitalocean/godo/blob/96c363fb60e090960d6ed4de5b5494b0e04c1c63/reserved_ips.go#L129) for reference.

2. Could you also rename `ReservedIPV6Resp` to `reservedIPV6Root` for consistency with existing code? :pray: 

3. Let's return `*ReservedIPV6` instead of `*ReservedIPV6Resp` for consistency with existing code? :pray: 

4. Let's try to reuse `reservedIPV6Root` in List as it is done [here](https://github.com/digitalocean/godo/blob/96c363fb60e090960d6ed4de5b5494b0e04c1c63/reserved_ips.go#L81)

Please, let me know when you are done. I'll be happy to take a look again :+1: